site/src/pages/components/menu.explainer.mdx

```diff
@@ -26,101 +26,111 @@ layout: ../../layouts/ComponentLayout.astro
 
 ## Introduction
 
-A menu is an essential way to group menu items together and provide users the ability to execute commands. There is an existing [`<menu>` element](https://html.spec.whatwg.org/multipage/grouping-content.html#the-menu-element), which represents a "toolbar" listing of commands. Though it is named "menu", it is not programmatically exposed like a "menu" or "menubar". There is also no native support to make listed commands look and behave cohesively. This proposal introduces new menu elements to give web developers more flexibility while following the [ARIA practices on menu and menubar](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/). Namely, our proposal includes adding `<menubar>`, `<menulist>`, and `<menuitem>` elements.
+An application menu is a common way to group actions together in the form of menu items, and provide
+users the ability to execute commands.
 
-Having native menu features will make it easier for web developers to add menulists, menubars and better configure the menu items to choose from. Furthermore, introducing these new features will expand on the baseline behaviors popover and anchor positioning provide.  New native menu elements will also have the correct implicit ARIA roles, states and properties as well as the expected keyboard navigation behaviors - ensuring consistent experiences for users on the platform they are engaging with.
+This proposal introduces the `<menubar>`, `<menulist>`, and `<menuitem>` elements to support
+application menus on the web. These menus will:
 
-Use cases:
+ - Represent commands as `<menuitem>`s, grouped together in a `<menulist>` popover or in-page
+   `<menubar>`.
+ - Represent state in checkable and radio-style menu items.
+ - Provide the correct implicit ARIA roles and keyboard interaction model out of the box.
+ - Give web developers default styles that create functional menus without having to re-invent
+   common popover and anchor positioning patterns, or re-implement ARIA patterns.
 
```

Comment:
I'd add a note, right here, that this explainer is for "application menus" and not "navigation menus" and point to the (perhaps placeholder) explainer for the other feature. Just to get that right up front.